### PR TITLE
Update parameter tests to remove regex expectations

### PR DESCRIPTION
**Pull request by SlayterDev**
https://app.asana.com/0/0/1204622926998046/f

### CODE REVIEW

**Review by kdzwinel — COMMENTED** [review]
test changes LGTM, but as discussed - we should add a test that DISALLOWS regexes to make sure that all platforms deprecate it

File: url-parameters/config_reference.json [hunk 1]
```diff
@@ -35,7 +35,8 @@
                     "mkt_tok",
                     "hmb_campaign",
                     "hmb_source",
-                    "hmb_medium"
+                    "hmb_medium",
+                    "bad[.*]"
```
Comment (kdzwinel) [inline url-parameters/config_reference.json:39]: can we make it simpler? I don't think all regex implementations are "full" (i.e. support all special characters). How about "bad." and then check if it doesn't affect "badx" param

Comment (kdzwinel) [reply url-parameters/config_reference.json:39]: Uh oh, or `rege.` and "regex" 👀

**Review by kdzwinel — APPROVED** [review]
LGTM, thank you!

**Comment by GuiltyDolphin** [comment]
This is causing the CI tests to fail now @kdzwinel @SlayterDev

**Comment by SlayterDev** [comment]
Seems to be a failure with the extension test which this fixes. The tests pass on the extension repo. Idk if these tests on this repo need an update or something.

**Comment by GuiltyDolphin** [comment]
I rebased on main and tests are passing for me again (#92). Possibly was outdated, or maybe flaky?

**Comment by GuiltyDolphin** [comment]
FYI this is causing the Android tests to fail when the action is run

**Pull request merged**